libs/gisday/platform/ngx/core/src/lib/pages/admin/pages/university/admin-add-university/admin-add-university.component.ts

```diff
@@ -21,6 +22,10 @@ export const formConfig = {
 export class AdminAddUniversityComponent extends BaseAdminAddComponent<University> {
   constructor(private fb1: FormBuilder, private universityService: UniversityService) {
     super(fb1, universityService);
-    this.formGroup = formConfig;
+    this.form = formExporter();
```

Comment:
Don't intiialize business logic in the constructor unless you need to. `form` does not need to be, I don't think. Can be moved to the OnInit.